tool/site/client/test-pack/prescriptions.ts

```diff
@@ -74,7 +74,7 @@ function createRepeatPrescribingPrescriptions(
   prescriptions: any[]
 ) {
   const repeatsAllowed = getNumberOfRepeatsAllowed(prescriptionRow)
-  for (let repeatsIssued = 0; repeatsIssued < repeatsAllowed; repeatsIssued++) {
+  for (let repeatsIssued = 1; repeatsIssued <= repeatsAllowed; repeatsIssued++) {
```

Comment:
Looks like we're sending in a message for each issue of the prescription, but that's not how eRD works. You just send in the first issue and Spine creates all of the subsequent issues automatically

Reply:
And for repeat prescribing, even if you're sending in issue 3 of 6 for example, you still tell Spine it's issue 1 of 1.

Reply:
as discussed, constraints from backend means this information is ignored, but suggest populating it as we are expecting in FHIR to be consistent with our docs and not expose inner workings of backend